itou/siaes/models.py

```diff
@@ -239,6 +208,22 @@ def display_name(self):
             return self.brand
         return self.name.capitalize()
 
+    @property
+    def is_active(self):
+        if self.kind not in Siae.ELIGIBILITY_REQUIRED_KINDS:
+            # GEIQ, EA... have no convention logic and thus are always active.
+            return True
+        if self.source != Siae.SOURCE_ASP:
+            # User created siaes and staff created siaes do not yet have convention logic.
+            return True
+        return self.convention and self.convention.is_active
```

Comment:
As-tu vraiment besoin des deux sentinelles avant `return self.convention and self.convention.is_active` ?

Reply:
Oui. Cf discussion plus haut.